Strings/src/main/java/net/avalith/StringsMethods.java

```diff
@@ -0,0 +1,126 @@
+package net.avalith;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class StringsMethods {
+
+    private Map<Character, Integer> romanValues;
+
+    public StringsMethods(){
+        this.romanValues = new HashMap<>();
+        romanValues.put('I', 1);
+        romanValues.put('V', 5);
+        romanValues.put('X', 10);
+        romanValues.put('L', 50);
+        romanValues.put('C', 100);
+        romanValues.put('D', 500);
+        romanValues.put('M', 1000);
+    }
+
+    public boolean isPalindrome(String string){
+        string = string.replaceAll("[^a-zA-Z]", "");
+        StringBuilder stringBuilder = new StringBuilder(string);
+        String aux = (stringBuilder.reverse()).toString();
+        if(aux.equalsIgnoreCase(string))
+            return true;
+        else
+            return false;
```

Comment:
```suggestion
        return aux.equalsIgnoreCase(string);
```
![DooJYJbXsAQZh7c](https://user-images.githubusercontent.com/60008637/82054938-1fbd9380-9696-11ea-966f-561bc01a7c4a.jpeg) Se obtiene el mismo resultado retornando el resultado del metodo que retorna `Boolean`

Reply:
No me di cuenta en el momento xd, perdón